src/Settings/GeneralSettings.ts

```diff
@@ -74,6 +74,16 @@ export function addGeneralSettings(plugin: BCPlugin, containerEl: HTMLElement) {
       })
     );
 
+  new Setting (generalDetails) 
+  .setName("show up fields in Juggl")
```

Comment:
Capitalize name. Also, make this a bit more clear in the description why this is needed. @SkepticMystic is this a good place to put this setting?